scss/main.scss

```diff
@@ -212,6 +201,26 @@ fieldset {
   }
 }
 
+a.btn {
```

Comment:
Does the selector need to be this specific (`a.btn.btn-tag`) for the right rules to kick in? I assume some rules should have inherited from `.btn` already?

Reply:
Yeah, it's mainly to _not_ inherit the `700` `font-weight` that's on `a.btn` from foundation button styles.

Reply:
Ah if that's the case can we just leave `font-weight: 400;` in `a.btn.btn-tag` and extract everything else to be in just `.btn-tag { }`? My concern is if everything is in `a.btn.btn-tag { }` now if we ever have to override any rules there in the future it'll get extra hard as the selector is very specific already.

Hopefully once major updates are done for Pulse we will have a chance to revisit all new styles introduced in Pulse and tidy up our SCSS to get rid of as many as overrides as we could.

Reply:
Trueee, yeah I can def do that. 👍

Reply:
> can we just leave font-weight: 400; in a.btn.btn-tag

btw, I put ☝️  the button override styles since it's overriding foundation styles.